src/components/PacksTable/PacksTable.tsx

```diff
@@ -25,6 +25,37 @@ const statusClassNames: Record<string, string> = {
   disabled: styles.disabled,
 };
 
+
+// Format the cloud type strings so they display properly 
+const formatCloudType = (type: string): string => {
+  const cloudTypeMapping: Record<string, string> = {
+    "aws": "AWS",
+    "eks": "EKS",
+    "vsphere": "vSphere",
+    "maas": "MaaS",
+    "gcp": "GCP",
+    "libvirt": "libvirt",
+    "openstack": "OpenStack",
+    "edge-native": "Edge-Native",
```

Comment:
```suggestion
    "edge-native": "Edge",
```

Let's just make this Edge. Edge native is not a term we use as it's a product that is only available to a specific customer. For all other customers, Edge is the correct term. This is confusing because the UI still has the term Edge native as an option when selecting a deployment infra provider.